tf2onnx/rewriter/leakyrelu_rewriter.py

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license.
+
+"""
+tf2onnx.rewrite - rewrite tensorflow subgraph to onnx leakyrelu op
+"""
+
+from tf2onnx.graph_matcher import OpTypePattern, GraphMatcher
+
+
+# pylint: disable=missing-docstring
+
+
+def rewrite_leakyrelu(g, ops):
+    if g.opset < 6:
+        return ops
+
+    pattern = \
+        OpTypePattern('Maximum', name='max', inputs=[
+            OpTypePattern('Mul', name='mul', inputs=[
+                OpTypePattern('Const', name='alpha'),
+                OpTypePattern('*', name='mul_input'),
+            ]),
+            OpTypePattern('*', name='max_input'),
+        ])
+
+    matcher = GraphMatcher(pattern, allow_reorder=True)
+    match_results = list(matcher.match_ops(ops))
+    for match in match_results:
+        max_node = match.get_op('max')
+        max_input_node = match.get_op('max_input')
+        mul_node = match.get_op("mul")
+        mul_input_node = match.get_op('mul_input')
+
+        max_input_edge_name = _find_edges_name_btw_nodes(max_input_node, max_node)
+        mul_input_edge_name = _find_edges_name_btw_nodes(mul_input_node, mul_node)
+        if max_input_edge_name == mul_input_edge_name:
```

Comment:
what does these two line check? 

        max_input_edge_name = _find_edges_name_btw_nodes(max_input_node, max_node) 
        mul_input_edge_name = _find_edges_name_btw_nodes(mul_input_node, mul_node)

Reply:
to see if one input of max and mul is same.